packages/documentation-v7/.storybook/preview.ts

```diff
@@ -42,7 +42,7 @@ const preview: Preview = {
           'Templates',
           'Utilities',
           'Misc',
-          ['Migration Guide', 'ChangeLog', 'Versions'],
+          ['Migration Guide', 'Change Log', 'Versions'],
```

Comment:
I'm more used to Changelog. I think both are right though, not sure.

Reply:
I renamed "Change Log" to "Changelog"